### PR TITLE
Currency:Added build-script, build_info.json and LICENSE for rake-nltk package.

### PR DESCRIPTION
**Pull request by abhijeetdandekaribm**
- Create build script rake-nltk_ubi_8.7.sh
- Create build_info.json
- Create LICENSE

### CODE REVIEW

**Comment by Sapana-Khemkar** [comment]
@seth-priya @nikhil-kalbande  we can consider this package for currency

File: r/rake-nltk/rake-nltk_ubi_8.7.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=rake-nltk
+PACKAGE_VERSION=a80f633
```
Comment (Sapana-Khemkar) [inline r/rake-nltk/rake-nltk_ubi_8.7.sh:22]: make PACKAGE_VERSION as input parameter with default value set. Something like PACKAGE_VERSION=${1:-a80f633}

File: r/rake-nltk/rake-nltk_ubi_8.7.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+pip3 install -r requirements.txt
+ret=0
+
+# Build step for a Python project
+python3 setup.py build || ret=$?
+
+if [ "$ret" -ne 0 ]
+then
+  echo "FAIL: Build failed."
+  exit 1
```
Comment (Sapana-Khemkar) [inline r/rake-nltk/rake-nltk_ubi_8.7.sh:67]: For currency return value should be as below
0 - success
1 - build failed
2 - test failed.
Please correct accordingly

File: r/rake-nltk/rake-nltk_ubi_8.7.sh (hunk 1 above)
Comment (abhijeetdandekaribm) [reply r/rake-nltk/rake-nltk_ubi_8.7.sh]: Made PACKAGE_VERSION as input parameter with default value set.

**Review by nikhil-kalbande — APPROVED** [review]

**Comment by nikhil-kalbande** [comment]
Looks good to add into currency. Merging it

File: r/rake-nltk/rake-nltk_ubi_8.7.sh (hunk 2 above)
Comment (saiNammi) [reply r/rake-nltk/rake-nltk_ubi_8.7.sh:57]: Hi @abhijeetdandekaribm , Why are you using multiple python versions?
yum -y update && yum install -y **python38 python38-devel python39 python39-devel python2 python2-devel python3 python3-devel**

**Pull request merged**